docs/usage/configuration-options.md

```diff
@@ -1166,6 +1166,18 @@ If the value starts with `http(s)` then it will only match against URLs which st
 Otherwise, it will be matched by checking if the URL's hostname matches the `matchHost` directly or ends with it.
 When checking the end of the hostname, a single dot is prefixed to the value of `matchHost`, if one is not already present, to ensure it can only match against whole domain segments.
 
+The `matchHost` URL must be the same as the `registryUrl` set in `.npmrc`, or you'll get authentication issues when the artifacts are updated when yarn or npm runs.
+
+```json
+{
+  "hostRules": [
+    {
+      "matchHost": "https://gitlab.myorg.com/api/v4/packages/npm/"
+    }
+  ]
+}
+```
```

Comment:
```suggestion
```

The above would correspond with a `.npmrc` containing `registry=https://gitlab.myorg.com/api/v4/packages/npm/`.

```